gammapy/data/event_list.py

```diff
@@ -379,6 +380,45 @@ def filter_circular_region(self, region):
 
         return mask
 
+    def select_region(self, region, wcs):
+        """Select events in circular regions.
+
+        Parameters
+        ----------
+        region : `~regions.SkyRegion` or str
+            Sky region or string defining a sky region
+        wcs : `~astropy.wcs.WCS`
+            the world coordinate system transformation to assume
+
+        Returns
+        -------
+        event_list : `EventList`
+            Copy of event list with selection applied.
+        """
+        mask = self.filter_region(region, wcs)
+        return self.select_row_subset(mask)
+
+    def filter_region(self, region, wcs):
+        """Create selection mask for event in given circular regions.
```

Comment:
"event in given circular regions" -> "events in given region."

Reply:
done